src/bflib_inputctrl.cpp

```diff
@@ -424,6 +445,104 @@ TbBool LbWindowsControl(void)
     while (SDL_PollEvent(&ev)) {
         process_event(&ev);
     }
+
+    // Poll controller state
+    if (controller != NULL) {
+        // Map controller buttons to keyboard keys
+        lbKeyOn[KC_W] = SDL_GameControllerGetButton(controller, SDL_CONTROLLER_BUTTON_DPAD_UP);
```

Comment:
best to not map this to WASD, but to arrow keys. Arrow keys also control camera, but cannot be rebound. WASD is very qwerty specific.